utils/ffmpegvideodecode/ffmpeg_video_dec.cpp

```diff
@@ -452,9 +460,29 @@ int FFMpegVideoDecoder::HandlePictureDecode(RocdecPicParams *pPicParams) {
 
     if (no_multithreading_) {
         DecodeAvFrame(av_pkt, dec_frames_[av_frame_cnt_]);
+        int num_frames_to_display = decoded_pic_cnt_;
+        while (num_frames_to_display) {
+            RocdecParserDispInfo dispInfo = { 0 }; // don't care about this as this will be igonored
```

Comment:
This comment is confusing here. Please either remove it or explain why it is being ignored?
also remove the extra space `{ 0 }` -> `{0}`